xchange-poloniex2/pom.xml

```diff
@@ -25,5 +25,10 @@
             <artifactId>service-netty</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>28.2-jre</version>
```

Comment:
The version shou;ld be removed. It's already under `dependencyManagement` in the parent POM.

Reply:
Done.